pkg/metainfo/kvmetainfo/buckets.go

```diff
@@ -222,9 +222,9 @@ func bucketFromMeta(ctx context.Context, bucketName string, m objects.Meta) (out
 	rs := &out.RedundancyScheme
 
 	out.Attribution = m.UserDefined["attribution-to"]
-	applySetting("path-enc-type", 16, func(v int64) { out.PathCipher = storj.Cipher(v) })
+	applySetting("path-enc-type", 16, func(v int64) { out.PathCipher = storj.CipherSuite(v) })
 	applySetting("default-seg-size", 64, func(v int64) { out.SegmentsSize = v })
-	applySetting("default-enc-type", 32, func(v int64) { es.CipherSuite = storj.Cipher(v).ToCipherSuite() })
+	applySetting("default-enc-type", 32, func(v int64) { es.CipherSuite = storj.CipherSuite(v) })
```

Comment:
are there backward compatibility concerns? like do old buckets need to keep working?